src/_community/rules.md

```diff
@@ -46,14 +46,38 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted in Quilt community spaces must meet the following criteria:
+
+* Projects must not be malicious - which includes (but is not limited to) malware, mods that include backdoors, or 
+  mods that harass specific people or groups of people (for example, by automatically kicking them from servers with 
+  the mod installed)
+* Projects must not be sexualised, hateful or otherwise inappropriate in nature, and this also applies to all code 
```

Comment:
I think context would matter in this case, but I don't think the ruleset necessarily needs to start getting into the particulars of slur reclamation - even if the user didn't intend to be discriminatory, it could reasonably be considered inappropriate, but I would just ask the user to not do that rather than pull out the hammer

Reply:
Resolved, reasonable enforcement